From b6d18ca77f131cdcaa10d0eaa9d303399767edf6 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Wed, 28 Aug 2019 19:18:14 +0200
Subject: [PATCH] certdata2pem.py: use python3

Comments in that file imply it is already py3 compatible.

Upstream-Status: Pending
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
---
 mozilla/Makefile        | 2 +-
 mozilla/certdata2pem.py | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/mozilla/Makefile b/mozilla/Makefile
index 6f46118..f98877c 100644
--- a/mozilla/Makefile
+++ b/mozilla/Makefile
@@ -3,7 +3,7 @@
 #
 
 all:
-	python certdata2pem.py
+	python3 certdata2pem.py
 
 clean:
 	-rm -f *.crt
diff --git a/mozilla/certdata2pem.py b/mozilla/certdata2pem.py
index 0b02b2a..7d796f1 100644
--- a/mozilla/certdata2pem.py
+++ b/mozilla/certdata2pem.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 # vim:set et sw=4:
 #
 # certdata2pem.py - splits certdata.txt into multiple files
